src/rhash/hash.c

```diff
@@ -3982,6 +3994,10 @@ void rc_hash_initialize_iterator(struct rc_hash_iterator* iterator, const char*
         {
           iterator->consoles[0] = RC_CONSOLE_ELEKTOR_TV_GAMES_COMPUTER;
         }
+        else if (rc_path_compare_extension(ext, "tzx"))
+        {
+          iterator->consoles[0] = RC_CONSOLE_ZX_SPECTRUM;
```

Comment:
Looking at some of the "official" collections, most files seem to be `tzx`, `tap`, `z80`, and `scl`. You're not handling `z80` or `scl`. Should we? It looks like `z80` is a memory snapshot, which might be convenient for faster boots, but completely eliminates the possibility that it was dumped from a user's personal collection. I think we should probably ignore it. `scl` appears to be a version of `trd` that ignores unused sectors, so it should still provide a consistent dump.

The [libretro core](https://docs.libretro.com/library/fuse/#extensions) does support `scl`. It also supports `rzx` and `trd`. `rzx` appears to be a TAS type file, so we shouldn't consider it, but `trd` appears to be another disk format.

Are `csw` and `psx` uncommon or deprecated? They're unique enough that I'd be fine leaving them in, but then we should also consider adding `trd`.

| extension | bizhawk | fuse | include? |
| --- | ------- | ---- | -------- |
| csw | yes | no | maybe? |
| dsk | yes | no | probably |
| pzx | yes | no | maybe? |
| rzs | no | yes | no |
| scl | no | yes | maybe? |
| tap | yes | yes | yes |
| trd | no | yes |  maybe? |
| tzx | yes | yes | yes |
| wav | yes | no | maybe not? |
| z80 | no | yes | no |

Reply:
The BizHawk core doesn't handle z80 or scl. It might be desirable to include these anyways (fwiw, BizHawk doesn't use the iterator, as it already identifies the file at this point, this is just for convenience libretro)